api/controller/endpoints.py

```diff
@@ -15,13 +15,16 @@
 #  You should have received a copy of the GNU Affero General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from bson.objectid import ObjectId
```

Comment:
looks unused

Reply:
Can needs to be removed, I think it was automatically imported by vscode when I was testing some stuff

Also need to double check that the student api is searching for student_id and not "BAZ"